template/src/ontology/Makefile.jinja2

```diff
@@ -841,6 +869,17 @@ public_release:
 	$(GITHUB_RELEASE_PYTHON) --release $(TAGNAME) $(RELEASEFILES)
 {%- endif %}
 
+{% if 'obo' in project.edit_format %}
+.PHONY: normalize_obo_src
+normalize_obo_src: $(SRC)
+	$(OWLTOOLS) $< --merge-axiom-annotations -o -f obo $(TMPDIR)/NORM.obo && robot convert -i $(TMPDIR)/NORM.obo -o $(TMPDIR)/NORM.tmp.obo && mv $(TMPDIR)/NORM.tmp.obo $(SRC)
```

Comment:
We should probably use `$(ROBOT)` here instead of `robot`. This will allow the use of the XML catalog, and should automatically fix obophenotype/uberon#2169.

Reply:
Ooops. THANKS!